packages/core-snapshots/src/manager.ts

```diff
@@ -79,38 +79,45 @@ export class SnapshotManager {
         this.database.close();
     }
 
-    public async rollbackChain(height) {
-        const lastBlock = await this.database.getLastBlock();
-        const config = app.getConfig();
-        const maxDelegates = config.getMilestone(lastBlock.height).activeDelegates;
+    public async rollbackByHeight(height) {
+        if (!height) {
```

Comment:
What about negatives?

```suggestion
        if (!height || height < 0) {
```

Reply:
👍